DTS-STN/canadian-dental-care-plan
# AB#3017 -- Refining letters mocks to match the structure of what is returned by Interop/CCT

### Description
As per title.

Additional changes:
* Added sample CCT response for a PDF and removed the `test.pdf`
* Removed `referenceId` field - we will be using the `id` (the letter ID) instead. The `referenceId` may not be unique and won't be used at all in our application
* Unneeded creation of letters in mock DB have been removed in favour of returning a static sample JSON response - this is because the `userId` (but rather the `clientId`) won't ultimately be used to query the letters, which is what the mock DB currently does and I didn't want to over-complicate the mocking

Note that I have not added the related fields (like subscription key header) to actually connect to Interop. That will come in a future PR

### Related Azure Boards Work Items
[AB#3017](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3017).

### Screenshots (if applicable)
PDF:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/46b62ce0-e2d0-4947-8b05-447b3596f091)

### Checklist

### Test Instructions
Run the application locally and navigate to `/en/letters`. Click on a letter. A PDF resembling the screenshot above should be shown.
